doc/passw_hardening/sonic-passwh.yang

```diff
@@ -0,0 +1,91 @@
+module sonic-passwh {
+    yang-version 1.1;
+    namespace "http://github.com/Azure/sonic-passwh";
+	prefix passwh;
+
+    description "PASSWORD HARDENING YANG Module for SONiC OS";
+
+	revision 2021-10-12 {
+        description
+            "First Revision";
+    }
+    typedef feature_state {
+        type enumeration {
+			enum enabled;
+			enum disabled;
+		}
+	}
+
+    container sonic-passwh {
+		container PASSWH {
+			description "PASSWORD HARDENING part of config_db.json";
+			container POLICIES {
+				leaf state {
+					description "state of the feature";

```

Comment:
Not sure why do we need this field, can we have this entry itself only if we desire the password hardening?

Reply:
state - is for enabling or disabling the feature.
basically is meaning that the admin user can choose if he wants to use the password hardening feature or not.

Reply:
Not sure when we'll configure this table with state=disabled....we could very well not have this entry itself..right?